lab_1/main.py

```diff
@@ -11,7 +11,17 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    symbols = ['!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '-', '_', '=', '+', '[', '{', '}',
+               ']', ';', '<', ',', '>', '.', '?', '|', '\\', ':', ';', '"', "'",
+               '/', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '\t']
+    if isinstance(text, str):
```

Comment:
put all checks with None at the beginning
like if the argument is incorrect, return None

It'd be easier to read